Plugin~/WebRTCPlugin/GraphicsDevice/Vulkan/VulkanGraphicsDevice.cpp

```diff
@@ -55,9 +55,8 @@ bool VulkanGraphicsDevice::InitV() {
 //---------------------------------------------------------------------------------------------------------------------
 
 void VulkanGraphicsDevice::ShutdownV() {
-    VULKAN_SAFE_DESTROY_COMMAND_POOL(m_device, m_commandPool, m_allocator);
-    vkDestroyDevice(m_device, NULL);
```

Comment:
This is avoiding test failed on vulkan. https://github.com/Unity-Technologies/com.unity.webrtc/pull/227/commits/492204bbd29252e8cf73078afd70a9b23b6ad5f7
Is plugin native test passed?

Reply:
OK, I will check the native test.

Reply:
All native tests are passed on Windows.
Which platform can we reproduce the bug?